system/greenhouse-ccloud/templates/thanos-pluginpreset.yaml

```diff
@@ -16,6 +16,11 @@ spec:
           paths:
             - path: /
               pathType: ImplementationSpecific
+    - name: thanos.query.ingress.grpc.tls
+      value:
+        - hosts:
+            - "thanos-grpc.{{ $ingressDomain }}"
+          secretName: {{ printf "tls-thanos-grpc-%s" $ingressDomain | replace "." "-" }}
```

Comment:
do we really need to explicitly overwrite this here? Can't we go with a default of the host with respective domain?

Reply:
it is part of clustetOptionOveride to be able to configure it with its domain name. what do you mean by default of the host ?